.eslintrc.js

```diff
@@ -4,13 +4,14 @@
 // License text available at https://opensource.org/licenses/MIT
 
 module.exports = {
-  extends: ['@loopback/eslint-config'],
+  extends: ['@loopback/eslint-config', 'plugin:cypress/recommended'],
   overrides: [
     {
       files: ['**/*.js'],
       rules: {
         '@typescript-eslint/no-unused-vars': 'off',
         '@typescript-eslint/no-misused-promises': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
Is this intentional?

Reply:
Correct. express-eslint seems to trigger floating promises linting issues on JQuery UI files

Reply:
Would it be better to add `// eslint-disable-next-line @typescript-eslint/no-floating-promises` to `public/ui.js:L10` since it is just the one error?

Reply:
Let me have a look

Reply:
Works will update